app/lib/system/redis_pool.rb

```diff
@@ -6,11 +6,12 @@ module System
   # RedisPool a simple wrapper around Redis with connection pooling
   class RedisPool
 
+    # @param [Hash] config - configuration, as in config/redis.yml
+    # @return System::RedisPool
```

Comment:
just a nit 
```suggestion
```

it is never relevant what initializers return, they are called internally